interface/main/main_title.php

```diff
@@ -87,6 +93,10 @@ function showhideMenu() {
             <a href='' class="css_button_small" style="margin:0px;vertical-align:top;display:none;" id='clear_active' onClick="javascript:parent.left_nav.clearactive();return false;">
             <span><?php echo htmlspecialchars( xl('CLEAR ACTIVE PATIENT'), ENT_QUOTES); ?></span></a>
     </td>
+    <td style="vertical-align:text-bottom;">
+        <a href='' class="css_button_small" style="margin:0px;vertical-align:top;display:none;" id='clear_active_group' onClick="javascript:parent.left_nav.clearactive();return false;">
+            <span><?php echo htmlspecialchars( xl('CLEAR ACTIVE THERAPY GROUP'), ENT_QUOTES); ?></span></a>
+    </td>
 </tr>
 <?php } //end of acl_check('patients','demo','',array('write','addonly') if ?>
 
```

Comment:
Who is the "tabs developer" you speak of ?
An option would be to turn it off when tabs UI is being used, but this is not ideal since most users will be using the tabs UI.